tests/t_server_null_stress.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Run this stress test as root to avoid sudo authorization from timing out.
+
+count=1
+
+. ./t_server_null_default.rc
+
+export pid_files=""
+for SUF in $TEST_SERVER_LIST
+do
+    eval server_name=\"\$SERVER_NAME_$SUF\"
+    pid_files="${pid_files} ${srcdir}/${server_name}.pid"
```

Comment:
```suggestion
    pid_files="${pid_files} ./${server_name}.pid"
```
Not really necessary to support out-of-tree here, but srcdir still doesn't make sense here.